Add Windows metrics in containerd integration (#10292)
DataDog/integrations-core: containerd/README.md

See [metadata.csv][4] for a list of metrics provided by this Integration.

This integration works on Linux and Windows, but some metrics are OS dependent.

> - Linux-specific: `containerd.hugetlb.*`, `containerd.blkio.*`, `containerd.cpu.throttled.periods`, and all the `containerd.mem.*` ones except these 3: `containerd.mem.commit`, `containerd.mem.commit_peak`, and `containerd.mem.private_working_set`.

Reviewer: ```suggestion
- Linux-specific: `containerd.hugetlb.*`, `containerd.blkio.*`, `containerd.cpu.throttled.periods`, and all the `containerd.mem.*` metrics except: `containerd.mem.commit`, `containerd.mem.commit_peak`, and `containerd.mem.private_working_set`.
```

See [metadata.csv][5] for a list of metrics provided by this integration.

> This integration works on Linux and Windows, but some metrics are OS dependent.
> - Linux-specific: `containerd.hugetlb.*`, `containerd.blkio.*`, `containerd.cpu.throttled.periods`, and all the `containerd.mem.*` metrics except: `containerd.mem.commit`, `containerd.mem.commit_peak`, and `containerd.mem.private_working_set`.
> - Windows-specific: `containerd.mem.commit`, `containerd.mem.commit_peak`, `containerd.mem.private_working_set`, and `containerd.storage.*`.
>  

Reviewer 2: If more OS dependent metrics are added, it may get hard to maintain in both the `README` and `metadata.csv`.
```suggestion
This integration works on Linux and Windows, but some metrics are OS dependent. Look at `metadata.csv` for the list of OS dependent metrics. 
 
```

Author: That's a good point. i've committed your suggestion.
